README.md

```diff
@@ -32,7 +32,9 @@ on which you want to lint
 
 # How to set up a generic Raspberry Pi environment (for a Brave Hub or the pairing tool):
 
-1. use Balena Etcher to flash the SD card with Raspbian Buster
```

Comment:
I like using Etcher - can we say something like 'use Etcher or RPi Imager to flash the SD card'